src/auth.ts

```
@@ -0,0 +1,61 @@
+                const payload: ApiResponse<LoginResponse> = await response.json();
+
+                if('error' in payload) {
+                    throw new Error(payload.error);
+                }
+                // Return user object to store in JWT
+                return {
+                    id: payload.user._id,
+                    user: payload.user,
+                    accessToken: payload.token,
+                }
+            }
```

ESLint

[error] Missing semicolon.

(semi)